tests/platform_tests/counterpoll/counterpoll_helper.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from tests.platform_tests.counterpoll.counterpoll_constants import CounterpollConstants
+
+
+class ConterpollHelper:
+    @staticmethod
+    def get_counterpoll_show_output(duthost):
+        counterpoll_show = duthost.command(CounterpollConstants.COUNTERPOLL_SHOW)
+        return counterpoll_show[CounterpollConstants.STDOUT]
+
+    @staticmethod
+    def get_available_counterpoll_types(duthost):
```

Comment:
This is a very basic function, which can list all the subcommands of a given CLI. I would suggest contribute this function to file tests/common/devices/sonic.py like:
```python
def list_subcommands(self, cmd):
    pass
```